sympy/core/symbol.py

```diff
@@ -201,21 +201,77 @@ def numbered_string_incr(s, start=0):
 
 class Symbol(AtomicExpr, Boolean):
     """
+    Symbols are variables, and like all other variables, they need to
+    be assigned before you can use them.
+    SymPy :func:`Symbol` function returns just a single symbol taken from argument ``name`` whereas :func:`symbols` function can be used
+    to return a sequence of symbols.
+    Symbols can be used to build SymPy expressions which are immutable.
```

Comment:
Use your code editor to wrap these lines to a consistent length.

Reply:
> Use your code editor to wrap these lines to a consistent length.

@oscarbenjamin I guess you mean I should wrap these lines in such a way that number of lines can be reduced like now I wrapped it to line number 206.

Reply:
No that's definitely not what I meant. Read PEP 8:
https://www.python.org/dev/peps/pep-0008/#maximum-line-length

Long lines are hard to read in many situations so it is better to wrap them so that the code has a consistent line length like say 80 characters per line. This is not followed strictly in sympy but there is no reason not to do it for lines like this that are in a docstring.

Wrapping the above would give something like
```
    Symbols are variables, and like all other variables, they need to be
    assigned before you can use them. SymPy :func:`Symbol` function returns
    just a single symbol taken from argument ``name`` whereas :func:`symbols`
    function can be used to return a sequence of symbols.  Symbols can be used
    to build SymPy expressions which are immutable.
```
This isn't something that needs to be done manually because most code editors have a way to do this automatically like you select the lines and then ask the editor to wrap them.

Reply:
> This isn't something that needs to be done manually because most code editors have a way to do this automatically like you select the lines and then ask the editor to wrap them.

@oscarbenjamin I made the changes now.
Thank you so much for explaining in detail,
I am learning a lot from you : )